test/kixi/stats/distribution_test.cljc

```diff
@@ -255,6 +267,10 @@
                             (sut/binomial {:n n :p p})))
     (is (converges-to-mean? a
                             (sut/normal {:location a :scale (/ 1 r)})))
+    (is (converges-to-mean? (exp (+ a (* 0.5 b b)))
+                            (sut/lognormal {:location a :scale b})))
+    (is (converges-to-mean? (/ (* s (+ (/ 1 r) 0.01)) (- s 1)) ; :scale > 1
```

Comment:
Why `:scale > 1` when 0 < x <= 1 is valid? 🤔

Reply:
Thanks for the catch.  It should be `:shape > 1` for the mean to be finite.
I will fix the code.